### ISSUE TITLE
OrderedDict error on Python 2.6 in wallet_journal.py

### ISSUE THREAD

**Issue by Kyria** [madcowfred/evething#33]
Since you said "Python >=2.6 <3.0" in readme, I tried on my deb6 which only has python 2.6...

In evething / thing / views / wallet_journal.py, it looks like you have only 

```
    from collections import OrderedDict
```

instead of :  

```
    try:
        from collections import OrderedDict
    except ImportError:
        from ordereddict import OrderedDict
```

**Comment by Gillingham** [comment]
Was this not fixed in cde266b0523b7f6ae0b722c9e7aa76e06a1279e2 ?  If so please close it

**Comment by madcowfred** [comment]
Uhh sure looks like it was.
